terraform/team-repo/lambdas/start-sync-team/index.js

```diff
@@ -1,8 +1,9 @@
-const AWS = require('aws-sdk');
-const CodeBuild = new AWS.CodeBuild();
+const { CodeBuildClient, StartBuildCommand } = require("@aws-sdk/client-codebuild");
+
+const client = new CodeBuildClient(config);
```

Comment:
@Mark-Simulacrum you wrote this code in https://github.com/rust-lang/simpleinfra/issues/376#issuecomment-1876082858 (it was January 😅).

I saw that `config` is an uninitialized variable.
Do you think we can initialize `CodeBuildClient` without arguments? I.e. `new CodeBuildClient()`

Reply:
Yeah that should be fine.

Reply:
Thanks, I removed `config` 👍